### PR TITLE
7 users

### LINKED ISSUE #5: Add View/Controller Testing
Needs testing for the base structure

### LINKED ISSUE #7: Add Users
Use [devise](https://rubygems.org/gems/devise) to add users for editing/controlling books they have posted.

### PR DESCRIPTION
**Pull request by XanderGardner**
closes #7 
Also closes #5

### CODE REVIEW

**Comment by XanderGardner** [comment]
Also closes #5

File: Gemfile [hunk 1]
```diff
@@ -55,6 +56,7 @@ group :test do
   gem 'capybara', '>= 3.26'
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
+  gem "shoulda-matchers", "~> 5.0"
```
Comment (eliotjordan) [inline Gemfile]: issue: Remove if not used.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -13,7 +13,10 @@
  */
 @import "bootstrap";
 
-@import "scaffolds";
+@import "general";
+@import "containers";
+
 @import "books";
```
Comment (eliotjordan) [inline app/assets/stylesheets/application.scss]: suggestion: Should remove `books` import if there is no styling to include. Same with `home`

File: app/assets/stylesheets/books.scss [hunk 3]
```diff
@@ -1,19 +1 @@
-h2 {
```
Comment (eliotjordan) [inline app/assets/stylesheets/books.scss:1]: suggestion: Since this is an empty file, I would just delete it.

**Review by eliotjordan — DISMISSED** [review]
Great work. 💯

**Review by eliotjordan — APPROVED** [review]

**Pull request merged**